src/main/java/org/spongepowered/common/event/SpongeEventManager.java

```diff
@@ -453,7 +454,9 @@ public boolean post(Event event) {
 
     public boolean post(Event event, boolean allowClientThread) {
         return post(event, getHandlerCache(event).getListeners());
+    }
 
-
+    public boolean post(Event event, PluginContainer plugin) {
+        return post(event, getHandlerCache(event).getListeners().stream().filter(l -> l.getPlugin().equals(plugin)).collect(Collectors.toList()));
```

Comment:
Lots of nested methods here, can you multi-line them so if any one of them (I know, highly unlikely) are on a new line? Helps with line numbers in stack traces for possible errors in the future.

Reply:
I can break the stream call up onto separate lines.